packages/tailwind/src/tailwind-presets.ts

```diff
@@ -52,81 +54,71 @@ export const kaizenTailwindTheme: KaizenTailwindTheme = {
     "family-display": [`${defaultTheme.typography.display0.fontFamily}`],
   },
   fontSize: {
-    "size-display": `${defaultTheme.typography.display0.fontSize}`,
```

Comment:
What's the rationale for removing `size-`? I thought the verbosity was clearer rather than guessing whether `font/text-display` came with anything more than just the size.

Reply:
Based on feedback we've been getting in the interviews it doesn't seem like this is needed, outside of tokens that may collide with each other (ie: font-weight and font-family)

Reply:
Interesting. I guess it just depends on if we choose to provide combined styles (eg. if `text-display` ends up being size, family, weight etc rolled in one). If we never plan to do so, then there won't be a conflict :)